main.go

```diff
@@ -158,6 +152,12 @@ Zero means no limits`,
 			DefValue:    3,
 			Description: "Number of times fetching deal data will be attempted before failing",
 		},
+		{
+			Name:        "deal-data-fetch-timeout",
+			DefValue:    "3h",
```

Comment:
`DefaultDataURIFetchTimeout` was changed from 3h to 10h when adding `est-download-speed`, now we set it back to 3h as a reasonable default.